src/schema-analyzer.ts

```diff
@@ -530,7 +549,7 @@ export class SchemaAnalyzer {
         }
 
         type.values.pushSome(
-          type.name === 'String' ? cropStringAt10kCharacters(value as string) : value
+          getCappedValue(type.bsonType, value)
```

Comment:
A few lines up, there's the line
```ts
          type.values = bsonType === 'String'
            ? Reservoir(100) : Reservoir(10000);
```
Do we want to do that for binary and code as well?